### PR TITLE
Fix .NET SourceLink and warnings in NuGet package

### PR DESCRIPTION
**Pull request by JamesNK**
* Include untracked sources in NuGet packages
* Create NuGet package using a deterministic build
* Fix NuGet warning by updating license URL to license expression
* Update .NET SDK so reproducable build metadata is added to package

Below is a comparison in NuGet Package Explorer.

**Before:**

image

**After: (note, package built locally on my PC so missing signatures from Google and NuGet)**

image

### CODE REVIEW

File: kokoro/linux/dockerfile/test/csharp/Dockerfile
Comment (jtattermusch) [inline kokoro/linux/dockerfile/test/csharp/Dockerfile:36]: is this required? I will need to rebuild the base dockerfile if so for the tests to pass.

**Review by jtattermusch — APPROVED** [review]
LGTM once the tests pass (and I'll need to push the docker image in order to make that happen I think).

File: kokoro/linux/dockerfile/test/csharp/Dockerfile
Comment (JamesNK) [reply kokoro/linux/dockerfile/test/csharp/Dockerfile:36]: Yes. SDK version needs to be 5.0.200 or later to embed all the required metadata on build.

**Comment by jtattermusch** [comment]
Look like on windows, there's again a problem with not being able to download nuget packages (due to old certs?):

```
T:\src\github\protobuf\csharp>dotnet restore src/Google.Protobuf.sln 


  Determining projects to restore...


T:\src\github\protobuf\csharp\src\Google.Protobuf.Benchmarks\Google.Protobuf.Benchmarks.csproj : error NU1101: Unable to find package BenchmarkDotNet. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf\Google.Protobuf.csproj : error NU1102: Unable to find package NETStandard.Library with version (>= 1.6.1) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf\Google.Protobuf.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 1.6.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf\Google.Protobuf.csproj : error NU1101: Unable to find package Microsoft.SourceLink.GitHub. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf\Google.Protobuf.csproj : error NU1101: Unable to find package Microsoft.NETFramework.ReferenceAssemblies. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf\Google.Protobuf.csproj : error NU1101: Unable to find package System.Memory. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf\Google.Protobuf.csproj : error NU1102: Unable to find package NETStandard.Library with version (>= 2.0.3) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf\Google.Protobuf.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 1.6.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf\Google.Protobuf.csproj : error NU1101: Unable to find package System.Runtime.CompilerServices.Unsafe. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


  Failed to restore T:\src\github\protobuf\csharp\src\Google.Protobuf\Google.Protobuf.csproj (in 186 ms).


T:\src\github\protobuf\csharp\src\Google.Protobuf.JsonDump\Google.Protobuf.JsonDump.csproj : error NU1101: Unable to find package Microsoft.NETCore.App. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.JsonDump\Google.Protobuf.JsonDump.csproj : error NU1101: Unable to find package System.Memory. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.JsonDump\Google.Protobuf.JsonDump.csproj : error NU1101: Unable to find package System.Runtime.CompilerServices.Unsafe. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


  Failed to restore T:\src\github\protobuf\csharp\src\Google.Protobuf.Benchmarks\Google.Protobuf.Benchmarks.csproj (in 216 ms).


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test.TestProtos\Google.Protobuf.Test.TestProtos.csproj : error NU1101: Unable to find package Microsoft.NETFramework.ReferenceAssemblies. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test.TestProtos\Google.Protobuf.Test.TestProtos.csproj : error NU1101: Unable to find package System.Memory. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test.TestProtos\Google.Protobuf.Test.TestProtos.csproj : error NU1102: Unable to find package NETStandard.Library with version (>= 1.6.1) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test.TestProtos\Google.Protobuf.Test.TestProtos.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 1.6.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test.TestProtos\Google.Protobuf.Test.TestProtos.csproj : error NU1102: Unable to find package NETStandard.Library with version (>= 2.0.3) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test.TestProtos\Google.Protobuf.Test.TestProtos.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 1.6.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test.TestProtos\Google.Protobuf.Test.TestProtos.csproj : error NU1101: Unable to find package System.Runtime.CompilerServices.Unsafe. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


  Failed to restore T:\src\github\protobuf\csharp\src\Google.Protobuf.JsonDump\Google.Protobuf.JsonDump.csproj (in 19 ms).


T:\src\github\protobuf\csharp\src\AddressBook\AddressBook.csproj : error NU1101: Unable to find package Microsoft.NETCore.App. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\AddressBook\AddressBook.csproj : error NU1101: Unable to find package System.Memory. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\AddressBook\AddressBook.csproj : error NU1101: Unable to find package System.Runtime.CompilerServices.Unsafe. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


  Failed to restore T:\src\github\protobuf\csharp\src\Google.Protobuf.Test.TestProtos\Google.Protobuf.Test.TestProtos.csproj (in 4 ms).


  Failed to restore T:\src\github\protobuf\csharp\src\AddressBook\AddressBook.csproj (in 2 ms).


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : warning NU1603: Google.Protobuf.Test depends on Microsoft.NET.Test.Sdk (>= 15.5.0) but Microsoft.NET.Test.Sdk 15.5.0 was not found. An approximate best match of Microsoft.NET.Test.Sdk 15.7.0 was resolved. [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : warning NU1603: Microsoft.TestPlatform.ObjectModel 15.7.0 depends on System.Diagnostics.TraceSource (>= 4.0.0) but System.Diagnostics.TraceSource 4.0.0 was not found. An approximate best match of System.Diagnostics.TraceSource 4.3.0 was resolved. [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : warning NU1603: Google.Protobuf.Test depends on Microsoft.NET.Test.Sdk (>= 15.5.0) but Microsoft.NET.Test.Sdk 15.5.0 was not found. An approximate best match of Microsoft.NET.Test.Sdk 15.7.0 was resolved. [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : warning NU1603: Microsoft.TestPlatform.ObjectModel 15.7.0 depends on System.Diagnostics.TraceSource (>= 4.0.0) but System.Diagnostics.TraceSource 4.0.0 was not found. An approximate best match of System.Diagnostics.TraceSource 4.3.0 was resolved. [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1101: Unable to find package Microsoft.NETCore.App. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1101: Unable to find package System.Memory. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1101: Unable to find package System.Runtime.CompilerServices.Unsafe. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1101: Unable to find package NUnit. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1101: Unable to find package NUnit3TestAdapter. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Runtime.Loader with version (>= 4.0.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.0-beta-23123 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Xml.XPath.XmlDocument with version (>= 4.0.1) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Diagnostics.Process with version (>= 4.1.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.0-beta-23123 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Threading.Thread with version (>= 4.0.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.0-beta-23123 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package Microsoft.NETCore.Platforms with version (>= 1.1.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 1.0.2 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package runtime.native.System with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Collections with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.11 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Diagnostics.Debug with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.11 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Globalization with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.11 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Resources.ResourceManager with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 2 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.1 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Runtime with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 4 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.1.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Runtime.Extensions with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.1.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102: Unable to find package System.Threading with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.11 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1101: Unable to find package NUnit. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1101: Unable to find package NUnit3TestAdapter. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1101: Unable to find package Microsoft.NETFramework.ReferenceAssemblies. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1101: Unable to find package System.Memory. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1101: Unable to find package Microsoft.NETCore.App. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1101: Unable to find package System.Runtime.CompilerServices.Unsafe. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Runtime.Loader with version (>= 4.0.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.0-beta-23123 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Xml.XPath.XmlDocument with version (>= 4.0.1) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Diagnostics.Process with version (>= 4.1.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.0-beta-23123 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Threading.Thread with version (>= 4.0.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.0-beta-23123 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package Microsoft.NETCore.Platforms with version (>= 1.1.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 1.0.2 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package runtime.native.System with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 1 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Collections with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.11 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Diagnostics.Debug with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.11 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Globalization with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.11 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Resources.ResourceManager with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 2 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.1 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Runtime with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 4 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.1.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Runtime.Extensions with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.1.0 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102: Unable to find package System.Threading with version (>= 4.3.0) [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj : error NU1102:   - Found 3 version(s) in Microsoft Visual Studio Offline Packages [ Nearest version: 4.0.11 ] [T:\src\github\protobuf\csharp\src\Google.Protobuf.sln]


  Failed to restore T:\src\github\protobuf\csharp\src\Google.Protobuf.Conformance\Google.Protobuf.Conformance.csproj (in 7.44 sec).


  Failed to restore T:\src\github\protobuf\csharp\src\Google.Protobuf.Test\Google.Protobuf.Test.csproj (in 7.52 sec).
```

**Comment by JamesNK** [comment]
There is this - https://github.com/dotnet/announcements/issues/180 - but it doesn't impact Windows. And this PR updates the SDK version to 5.0.202 anyway, which supposedly fixes this problem.

**Comment by jtattermusch** [comment]
I'll try rerunning the tests. (Btw, we fixed the internal problem with the protobuf test result links not being accessible externally, so you should now be able to see the tests results again yourself).

**Comment by jtattermusch** [comment]
I restarted to tests to see if the build problems (e.g. https://source.cloud.google.com/results/invocations/203a7546-7336-49c8-a636-d1661ffd81dd/log) persist.

**Comment by jtattermusch** [comment]
Is there a way to revive this and make the tests pass? At this point the PR's breaking tests so we can't merge it.
If we don't know what to do about this we might as well close since there's no value in having this PR open forever without being able to merge it.

**Comment by JamesNK** [comment]
@jtattermusch PR rebased. It should now pass.

**Pull request merged**
